meetings/2021-06-09.md

```diff
@@ -0,0 +1,137 @@
+- My authorization agent can help me manage the access granted to a specific application
+- Why would I just give all access to Acme project?
+- What you want is Acme project to say I have no access
+- Acme looks in the WebID and sees who my authorisation agent is
+- Acme project sends me to my authorisation agent and express the kind of data they need authorisation to (Project Management data for example) Then asks which projects? all/work...
+- I want only work projects access for acme project and the app gets back the list of things they have access to
+
+That's an oversimplyfied example of what UMA does.
+It would facilitate that kind of fined grained access.
+Which is a pretty necessary pattern.
+You can build upon it.
+We currently haven't used UMA, we just made an authorization agent with sanity checking.
+Currently UMA is supported by several IdPs and the benefit of investigating it is that we've talked about IdP potentially doing that.
+
+Matthieu: In short the authorization agent helps you manage fine grained access for specific agents.
+
+See:
+![](https://i.imgur.com/msdMwES.png)
+
+See also the [data interoperability walkthrough videos](https://github.com/solid/data-interoperability-panel#solid-application-interoperability):
+  * [Application Interoperability Walkthrough - Part 1](https://solid-interop-public.s3.amazonaws.com/application-interoperability-walkthrough-part-1.mp4) (VIDEO)
+  * [Application Interoperability Walkthrough - Part 2](https://solid-interop-public.s3.amazonaws.com/application-interoperability-walkthrough-part-2.mp4) (VIDEO)
+
+Henry: The authorization agent says this app can access this kind of resources. So we're back to something like we could use owl. For example everything tagged banking...
+
+Justin: What we do in Interop is organise data by type using shapetrees. So when you get access, you essentially get access to a type within a scope. So the decision I make is an access grant and can be stored in my Pod with other grants. It's a pretty big subject for 8mn. This is a layer above it.
+
+Access Grant for Acme Project:
+![](https://i.imgur.com/bfngw64.jpg)
+
+Justin: This is what the authorisation agent create.
+We just show the grants here, but the access needs groups stores what was requested and what was granted.
+You can actually do a diff and store it if you want more access.
+
+Henry: I think it's close to my proposal on my launcher app.
+
+Justin: Wasn't it proxying all the time?
+
+Henry: In an ideal world, (simple), the app shouldn't get access to the internet without going through the authorisation proxy. Imagine the app can only access web via your own managed proxy. You don't give it any cookies or certificates unless it goes through it. Now the proxy would have to know for each app which type of resources it has access to and what it can do with it. Perhaps some apps on your system and then you open when comfortable.
```

Comment:
```suggestion
Henry: In an ideal world, (simple), the app shouldn't get access to the internet without going through the authorization proxy. Imagine the app can only access web via your own managed proxy. You don't give it any cookies or certificates unless it goes through it. Now the proxy would have to know, for each app, which type of resources it has access to and what it can do with it. Perhaps some app's on your system and then you open when comfortable.
```